roles/server/tasks/composer.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: install composer
+  shell: curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
```

Comment:
~~Is the created file executable?~~ Looks like the installer sets the permissions for us.

```php
if ($errorHandler->message) {
    out('The download failed repeatedly, aborting.', 'error');
    exit(1);
}

chmod($file, 0755);

if (!$quiet) {
    out(PHP_EOL."Composer successfully installed to: " . $file, 'success', false);
    out(PHP_EOL."Use it: php $installPath", 'info');
}
```

No flags set, rwx for owner, rx for group and world.